src/widgets/dialogs/UserInfoPopup.cpp

```diff
@@ -33,7 +33,7 @@
 const QString TEXT_VIEWS("Views: %1");
 const QString TEXT_FOLLOWERS("Followers: %1");
 const QString TEXT_CREATED("Created: %1");
-const QString TEXT_TITLE("%1's Usercard");
+const QString TEXT_TITLE("%1's Usercard - %2");
```

Comment:
It might be a good idea to make some kind of indication to which one's the channel name and which is the user name.
A # like in irc might be the way to go.
```suggestion
const QString TEXT_TITLE("%1's Usercard - #%2");
```